depot/containerstore/dependencymanager.go

```diff
@@ -39,7 +39,7 @@ func (bm *dependencyManager) Stop(logger lager.Logger) {
 	}
 }
 
-func (bm *dependencyManager) DownloadCachedDependencies(logger lager.Logger, mounts []executor.CachedDependency, streamer log_streamer.LogStreamer) (BindMounts, error) {
+func (bm *dependencyManager) DownloadCachedDependencies(logger lager.Logger, mounts []executor.CachedDependency, logConfig executor.LogConfig, metronClient loggingclient.IngressClient) (BindMounts, error) {
```

Comment:
Looks like we are not limiting downloading cached dependencies? Don't we want to limit them?

Reply:
I wouldn't think it'd be a major concern. All we're outputting is "Downloading x" and "downloaded x" or "failed to download x". I wouldn't think it'd be a major concern with regards to volume? 

Maybe I'm misunderstanding what dependency manager is, but it also means we don't create two log-streamers which is vital for metric tracking. I also think broadly speaking, we shouldn't have to worry about the small amount of tracking logs that exist(starting container, stopping container, etc). It's all about limiting the possibly volumous user generated logs.